docs/platforms/react-native/overhead/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: SDK Overhead
+description: "Learn about Sentry's React Native SDK overhead."
+sidebar_order: 7500
+---
+
+Adding new features or dependencies to your app incurs additional costs on resources - CPU time, memory usage, and network bandwidth, among other things. Sentry SDKs are no different. This document adds transparency to the possible overhead that using our SDK can add, and help you find the feature set and configurations that work best for you.
+
+## General Approach
+
+The SDK is designed to have minimal to no impact on user experience. To achieve this, we utilize an array of tools to continuously measure and optimize the performance of our implementations.
+
+We also employ various techniques to ensure we don't add strain on the system's resources along the hot path - on Mobile, this very often means that we offload processing steps, I/O, and other things to a background thread, or we postpone processing to a later time if possible.
```

Comment:
nit: This is fine as it is, but I suggest splitting it into two sentences, as I found it confusing on the first read. 
```suggestion
We also employ various techniques to ensure we don't add strain on the system's resources along the hot path. On Mobile, this very often means that we offload processing steps, I/O, and other things to a background thread, or we postpone processing to a later time if possible.
```